packages/cli/src/commands/spaces/outbound-rules/index.ts

```diff
@@ -0,0 +1,51 @@
+import {Command, flags} from '@heroku-cli/command'
+import {Args, ux} from '@oclif/core'
+import heredoc from 'tsheredoc'
+import {displayRules, displayRulesAsJSON} from '../../../lib/spaces/outbound-rules'
+import * as Heroku from '@heroku-cli/schema'
+import {SpaceCompletion} from '@heroku-cli/command/lib/completions'
+
+export default class Index extends Command {
+  static description = heredoc(`
+    list Outbound Rules for a space
+    Outbound Rules are only available on Private Spaces.
+
+    Newly created spaces will have an "Allow All" rule set by default
+    allowing all egress dyno traffic outside of the space.  You can
+    remove this default rule to completely stop your private dynos from
+    talking to the world.
+  `)
+
+  static aliases = ['outbound-rules']
+  static hidden = true;
```

Comment:
Besides, the semicolon, 😆 , do we know why this command has the hidden attribute? Does it make sense to keep it hidden?

Reply:
i wondered that myself. i looks like a lot of the `spaces` commands are hidden.. probably worth talking to anush about.